modules/ROOT/pages/to-create-an-asset.adoc

```diff
@@ -27,6 +27,26 @@ This illustration summarizes how each asset type (in green) appears in Exchange:
 
 image::ex2-exchange-assets.png[Illustration - How Assets appear in Exchange]
 
+== Asset Limits
+
+Exchange uses an asset limit to prevent accidental or intentional denial of service by uploading too many assets.
+
+The limit is 500 assets for master organizations with trial accounts, and 100 000 (one hundred thousand) assets for other master organizations.
+
+This asset count does not include soft deleted assets, hard deleted assets, or generated assets such as xref:to-deploy-using-rest-connect.adoc[Mule 3 and Mule 4 connectors generated automatically from APIs].
```

Comment:
They're defined at https://docs.mulesoft.com/exchange/to-delete-asset , but in this case I don't think the distinction is relevant. I've changed this to "This asset count does not include deleted assets or generated assets such as Mule 3 and Mule 4 connectors generated automatically from APIs." Is that good?

Reply:
Much better. But @mcampo & @LucasLudueno , can you please confirm the business logic, and make sure that Will's latest statement is correct?

Reply:
Just a question, should we write here the definition of what soft and hard delete assets means? Or should we write a link point to some page where the definition is?

And regarding the suggestion of Kuntal:

`hard deleted asset version`: Asset version that has been deleted totally from Exchange. That means that the version does not exist anymore in Exchange and everything that is part of the asset (like metadata, tags, and files). A user can delete an asset as hard-delete only if the asset has less than 7 days after it has been created (via API). After 7 days, it can not be deleted as hard-delete anymore. (From the Exchange UI, if the asset is between 7 days, the asset is deleted as hard-delete by default)

`soft deleted asset version`: Asset version that has been deleted from Exchange but the same GAV can not be reused anymore. That means that if a user tries to create an asset with the same GAV of a soft-deleted version, will receive an error. Also, metadata, tags, and files are deleted from Exchange. A user can delete an asset as soft-delete anytime (via API). (From the Exchange UI, if the asset has more than 7 days, the asset is deleted as soft-delete)

Reply:
Yes it is correct @kuntald

Reply:
Looks good. I'm indifferent whether we skip the soft/hard mention or keep it. But if we do keep it, then we should link to the definitions for each.

Reply:
Adding a reference to the delete page that Will has written is enough for me. I would only highlight the `soft delete` and `hard delete` terms in this doc (https://docs.mulesoft.com/exchange/to-delete-asset), but it is only an opinion

Reply:
I agree, better to highlight those. Made that change.